Project.toml

```diff
@@ -16,6 +16,7 @@ ImageMagick = "6218d12a-5da1-5696-b52f-db25d2ecc6d1" # seems to be needed for Ju
 Images = "916415d5-f1e6-5110-898d-aaa5f9f070e0"
 Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
 Requires = "ae029012-a4dd-5104-9daa-d747884805df"
+Revise = "295af30f-e4ad-537b-8983-00126c2a3abe"
```

Comment:
Do you really need this? I think you just use this for your own debugging of the package. I think we don't need this for the actual package